hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -418,6 +426,7 @@ public int getSnapshotCacheSize() {
   public void invalidateCache() {
     if (snapshotCache != null) {
       snapshotCache.invalidateAll();
+      snapshotCache.close();
```

Comment:
I think it should be closed inside `OmSnapshotManager#close()` because invalidateCache is also called by [OzoneManager#installCheckpoint](https://github.com/apache/ozone/blob/9efbccea8aaf0aaa14686b53927a64c4d13118a0/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java#L3788) and I don't think we want to close scheduler in that case.

Reply:
done

Reply:
nit: I just realized if it would be better to call `snapshotCache.invalidateAll()` from inside of `snapshotCache.close()`.

Reply:
done